modular_skyrat/modules/liquids/code/liquid_systems/liquid_effect.dm

```diff
@@ -503,6 +513,32 @@
 	RegisterSignal(my_turf, COMSIG_ATOM_ENTERED, .proc/movable_entered)
 	RegisterSignal(my_turf, COMSIG_TURF_MOB_FALL, .proc/mob_fall)
 
+/**
+ * Handles COMSIG_PARENT_EXAMINE for the turf.
+ *
+ * Adds reagent info to examine text.
+ * Arguments:
+ * * source - the turf we're peekin at
+ * * examiner - the user
+ * * examine_text - the examine list
+ *  */
+/obj/effect/abstract/liquid_turf/proc/examine_turf(turf/source, mob/examiner, list/examine_text)
+	SIGNAL_HANDLER
+
+	// This should always have reagents if this effect object exists, but as a sanity check...
+	if(!length(reagent_list))
+		return
+
+	if(examiner.can_see_reagents()) //Show each individual reagent
+		examine_text += "<hr>"
```

Comment:
This should go after everything the scanner shows you, if you really want to have a line like this in the examine text (so move it to after line 540).

Reply:
Likewise, our codebase (skyrat edit) places the HR before the atom reagent examine text as well.